openplacereviews-core/src/main/java/org/openplacereviews/osm/service/MergePlaceBot.java

```diff
@@ -211,6 +213,141 @@ protected void mergePlaces(OprMapCollectionApiResult res, MergeInfo info) {
 
 		}
 	}
+
+	private boolean closeDeletedPlace(OpObject deleted, List<OpObject> edited) {
+		if (wasDeletedMoreThanTenDaysAgo(deleted)) {
+			OprMapCollectionApiResult res = getDataReport(deleted);
+			if (res != null && res.geo.features() != null) {
+				List<Feature> features = res.geo.features();
+				List<Feature> placesToMerge = getNoDeletedPlaces(features);
```

Comment:
this all 3 methods could be combined in 1 method
getNondeletedClosedPlaces(res.geo.feature(), deleted)